lib/resources/majors.tsx

```diff
@@ -2395,135 +2395,157 @@ const bsBME: Major = {
-//         },
-//         {
-//           description:
-//             "Select one Biology Lab Course of the following:\n\t" +
-//             "AS.020.153 General Biology Laboratory I\n\tAS.020.154 General Biology Lab II\n\tAS.020.315 Biochemistry Project labs\n\t" +
-//             "\n\tAS.020.316 Cell Biology Lab\n\tAS.020.340 Developmental Genetics Lab\n\tAS.020.377 Comparative Physiology Lab\n\t" +
-//             "AS.250.253 Protein Engineering and Biochemistry Lab\n\tAS.250.254 Protein Biochemistry and Engineering Laboratory",
-//           required_credits: 1,
-//           criteria:
-//             "AS.020.153[C]^OR^AS.020.154[C]^OR^AS.020.315[C]^OR^AS.020.316[C]^OR^AS.020.340[C]^OR^AS.020.377[C]" +
-//             "^OR^AS.250.253[C]^OR^AS.250.254[C]",
-//         },
-//       ],
-//     },
-//     {
-//       name: "Quantitative",
-//       required_credits: 4,
-//       min_credits_per_course: 4,
-//       description: "Must complete the following courses in Quantitative.",
-//       criteria: "AS Quantitative[N]",
-//       fine_requirements: [
-//         {
-//           description:
-//             "Select one Quantitative Course of the following:\n\t110.106 Calculus I (Biology and Social Sciences)\n\t" +
-//             "110.108 Calculus I (Physical Sciences & Engineering)",
-//           required_credits: 4,
-//           criteria: "AS.110.106[C]^OR^AS.110.108[C]",
-//         },
-//       ],
-//     },
-//   ],
-// };
+const baPH: Major = {
+  degree_name: 'B.A. Public Health Studies',
+  abbrev: 'B.A. Public Health',
+  department: 'AS Public Health Studies',
+  total_degree_credit: 120,
+  wi_credit: 15,
+  url: 'https://e-catalogue.jhu.edu/arts-sciences/full-time-residential-programs/degree-programs/public-health-studies/public-health-studies-bachelor-arts/',
+  distributions: [
+    {
+      name: 'Public Health Studies Core Courses',
+      required_credits: 40,
+      min_credits_per_course: 1,
+      description:
+        'For more information please visit the' +
+        "<a href='https://e-catalogue.jhu.edu/arts-sciences/full-time-residential-programs/degree-programs/public-health-studies/public-health-studies-bachelor-arts/'>" +
+        'major degree requirement</a> section on the department website.',
+      criteria: 'AS Public Health Studies[D]',
+      fine_requirements: [
+        {
+          description:
+            'Required Courses: \n\t280.101 Introduction to Public Health (Fall & Spring)\n\t280.240 Research Methods in Public Health (Fall & Spring)\n\t' +
+            '280.335 The Environment and Your Health (Fall & Spring) \n\t280.340 Fundamentals of Health Policy & Management (Spring)\n\t' +
+            '280.345 Public Health Biostatistics (Fall)\n\t280.350	Fundamentals of Epidemiology (Fall & Spring)',
+          required_credits: 21,
+          criteria:
+            'AS.280.101[C]^OR^AS.280.240[C]^OR^AS.280.240[C]^OR^AS.280.335[C]^OR^AS.280.340[C]^OR^AS.280.345[C]^OR^AS.280.350[C]',
+        },
```

Comment:
why not make each course its own fine requirement? :3

Reply:
see bs CS for example 
the core classes are divided into gateay java, data structures, csf, etc...